### ISSUE TITLE
Not able to find the CADP dataset

### ISSUE THREAD

**Issue by PHDPeter** [ankitshah009/CarCrash_forecasting_and_detection#19]
Hi
I was unable to find the videos as part of the CADP dataset (https://drive.google.com/drive/folders/1ozfZbz8wRoNqKknC59IPXP_twKgmCOaA).
Could you please tell me where I might be able to download them from?

**Comment by PHDPeter** [comment]
I followed the code in dowbload_cadp.py but just got the error "Permission denied: '/media/tuananhn'"
I assume this means the data is not publicly available anymore?

**Comment by draxler1** [comment]
Yes. A particular GitHub repository does not have the dataset. You can download the CADP dataset from this site, however, all the videos are available in the frames format. You must convert according to your need.

`https://drive.google.com/file/d/1PDsYAsOEwZTcFVVt444wvhOTfjqwqTaY/view?usp=sharing`

ALL THE BEST :-)

**Comment by xingchenzhang** [comment]
> Yes. A particular GitHub repository does not have the dataset. You can download the CADP dataset from this site, however, all the videos are available in the frames format. You must convert according to your need.
> 
> `https://drive.google.com/file/d/1PDsYAsOEwZTcFVVt444wvhOTfjqwqTaY/view?usp=sharing`
> 
> ALL THE BEST :-)

Hi,

I have tried to download the dataset from your google drive (the link available on your website). I got more than 30 .zip files (something like 'Dataset-20210410T205413Z-027.zip'). Could you please let me know how to obtain the full dataset from these files? I tried to unzip one of them and I can find some mp4 videos. Are these the original videos you used?

Thanks a lot!
Xingchen

**Comment by MohammedMusadiq** [comment]
What about the annotations ? 
Did anyone get them?
